src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -37,19 +37,19 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new PhpFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('twig.php');
 
-        if (class_exists(\Symfony\Component\Form\Form::class)) {
+        if ($container->isAvailable('symfony/form', \Symfony\Component\Form\Form::class)) {
             $loader->load('form.php');
         }
 
-        if (class_exists(Application::class)) {
+        if ($container->isAvailable('symfony/console', Application::class)) {
```

Comment:
the Twig commands should be registered as soon as the symfony/console is installed, even if only a dev requirement. the commands are dev tools only anyway